styleguide/derek/examples/stories-interior.ejs

```diff
@@ -0,0 +1,49 @@
+<%- partial("../_partials/_ceiling") %>
+<%- partial("../_partials/_mainnav") %>
+<%- partial('../_partials/solutions/header-smallIMAC', { headline: 'Customer Stories'}) %>
```

Comment:
@amberRrucker,  The new github review system doesn't seem to hide messages once they've been addressed any more.  I just pulled the HTML directly into the template.  

https://github.com/rackerlabs/zoolander/pull/265/files#diff-a07e96d6486c531c335851c2ea60f322R13